src/compiler/parser-proc.bas

```diff
@@ -610,6 +640,22 @@ private function hCheckOpOvlParams _
 	end if
 #endmacro
 
+#macro hCheckParamIsUDT( proc, param, num )
+	'' param is same type as parent?
```

Comment:
This comment seems to be a copy-paste error

Reply:
yes, should be - param is UDT?

lol, which, now looking at the name of the macro, the comment seems to stating the obvious...